lib/strings.md

```diff
@@ -5,20 +5,20 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a String?
 
-Replace me with your response...
+A String is a variable that stores a sequences of characters.
 
 # What are some examples of information that would be Strings as opposed to some other data type?
 
-Replace me with your response...
+Words, sentences or any kind of text would be a String.  Also, input from a user is often saved as a String, even if it will be converted to another variable type later.
 
 # What is one way, using Ruby, to retrieve the 6th character in a String like `"Ada Lovelace"`? How about the 8th character? What happens if you try to retrieve the value of the _99th_ character (Or any character that doesn't exist)?
 
-Replace me with your response...
+"Ada Lovelace"[6] and "Ada Lovelace"[8].  string_name[n] will return the nth character of any String.  Attempting to retrieve a nonexistant character returns the value 'nil'.
```

Comment:
This explanation makes sense to me if you are thinking of `"A"` as the 0th character.